docs/data/pages.ts

```diff
@@ -585,10 +585,13 @@ const pages: MuiPage[] = [
                     subheader: 'Interfaces',
                     children: [
                       { pathname: '/x/api/charts/axis-config', title: 'AxisConfig' },
-                      { pathname: '/x/api/charts/bar-series-type', title: 'BarSeriesType' },
-                      { pathname: '/x/api/charts/line-series-type', title: 'LineSeriesType' },
-                      { pathname: '/x/api/charts/pie-series-type', title: 'PieSeriesType' },
-                      { pathname: '/x/api/charts/scatter-series-type', title: 'ScatterSeriesType' },
+                      { pathname: '/x/api/charts/bar-series', title: 'BarSeries' },
+                      { pathname: '/x/api/charts/funnel-series', title: 'FunnelSeries' },
+                      { pathname: '/x/api/charts/heatmap-series', title: 'HeatmapSeries' },
+                      { pathname: '/x/api/charts/line-series', title: 'LineSeries' },
+                      { pathname: '/x/api/charts/pie-series', title: 'PieSeries' },
+                      { pathname: '/x/api/charts/radar-series', title: 'RadarSeries' },
+                      { pathname: '/x/api/charts/scatter-series', title: 'ScatterSeries' },
```

Comment:
You could add them in the `pocs/public/_redirects` file such that links to previous page still work

```
/x/api/charts/scatter-series-type/ /x/api/charts/scatter-series/ 301
```

Reply:
As I was adding these, I realized that maybe we want to keep the old pages? 

The `SeriesType` is required for the `ChartDataProvider` as that component requires the `type` to be specified. 

Should we keep both?

Reply:
It feels a bit weird to have two API pages just because one property can be optional

Reply:
Yeah, true. I'll keep it as is, then